etcdctl/ctlv3/command/make_mirror_command.go

```diff
@@ -45,7 +48,8 @@ func NewMakeMirrorCommand() *cobra.Command {
 	}
 
 	c.Flags().StringVar(&mmprefix, "prefix", "", "Key-value prefix to mirror")
-	// TODO: add dest-prefix to mirror a prefix to a different prefix in the destination cluster?
+	c.Flags().StringVar(&mmdestprefix, "dest-prefix", "", "destination prefix to mirror a prefix to a different prefix in the destination cluster")
+	c.Flags().BoolVar(&mmremdestprefix, "rem-dest-prefix", false, "remove the key-value prefix in the destination cluster")
```

Comment:
@xiang90 i didn't understand what exactly you want me to change. Can you elaborate.
Thanks!